components/cli/vendor/github.com/docker/docker/pkg/mount/mounter_linux.go

```diff
@@ -29,8 +29,9 @@ func isremount(device string, flags uintptr) bool {
 
 func mount(device, target, mType string, flags uintptr, data string) error {
 	oflags := flags &^ ptypes
-	if !isremount(device, flags) {
-		// Initial call applying all non-propagation flags.
+	if !isremount(device, flags) || data != "" {
```

Comment:
this change is a result of not having #121 also vendor the changes to the `components/engine` dir